galaxy/main/celerytasks/user_notifications.py

```diff
@@ -259,13 +391,22 @@ def _get_preferences(users):
 import_status_template = '''Hello,
 
 This message is to notify you that a recent import of {content_name} on \
-Ansible galaxy has {status}.
+Ansible Galaxy has {status}.
 
 To see the import log, go to: {import_url}
 '''
 
 
-update_collection_template = '''Hello,
+collection_new_version_template = '''Hello,
+
+{namespace_name} has just released {content_name} version {version} on \
```

Comment:
We decided yesterday that any user facing views that we have should use the term 'role' and any author facing views should use the term 'repository', so the email and app notifications should probably reflect that.

Reply:
Updated for repository roles...
For user:
> One of the author's ... that you are following ... has just released a new __role__ ..."

For author:
> Someone has just submitted a new survey ... Your __repository__ now has a user rating of ..."